.github/workflows/merge-flow.yaml

```diff
@@ -160,7 +160,7 @@ jobs:
           SLACK_WEBHOOK_URL: ${{ secrets.slack-webhook-url }}
```

Comment:
should we make `required: true` for this secret?

Reply:
Nice catch, fixed it.